pkg/image/registry_options.go

```diff
@@ -0,0 +1,16 @@
+package image
+
+// RegistryOptions for the OCI registry provider.
```

Comment:
Currently this is supporting a little more than auth --the `InsecureSkipTLSVerify ` option is for the http transport beyond just auth (includes any http interaction). I'm up for any name change here, but wanted to point out the intended scope for this object.

Reply:
If a user can set TLS for any registry interaction, that may be tricky if supporting one registry that needs TLS and another one that doesn't. Are you implying that the TLS option shouldn't change and be global for any/all registries?

Reply:
good question, we could mimic docker more closely in this way --where they allow TLS verification configuration on a per-registry basis.

Reply:
After a little more thought, I think it makes sense to leave this as is for now... but we can add it later as an enhancement since it isn't mutually exclusive with a global option.